lib/generators/rspec/feature/feature_generator.rb

```diff
@@ -8,8 +8,9 @@ class FeatureGenerator < Base
 
       def generate_feature_spec
         return unless options[:feature_specs]
+        file_name = table_name.split(" ").collect(&:downcase).join("_")
```

Comment:
Since this is in a generator Rails, and Active Support, have already been loaded. Perhaps change this to the following to provide a wider range of character conversion support:

```ruby
file_name = table_name.parameterize.underscore
```

Thoughts?